daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/XMLTextInfosetOutputter.scala

```diff
@@ -144,11 +143,11 @@ class XMLTextInfosetOutputter private (writer: java.io.Writer, pretty: Boolean,
 
   override def startDocument(): Boolean = {
     writer.write("<?xml version=\"1.0\" encoding=\"UTF-8\" ?>")
```

Comment:
Honestly, I think it was just copied from some web site. Doing a little googling for XML declaration, the first few results have the space so that's probably where I got it. But if it keep going most places seem to not have a space. And all of our XML and TDML files do not have a space, so no space certainly seems like the right convention to use.

I would expect all tests shouldn't be comparing XML in a way that doesn't care about this space (I know our TDML runner does), so it doesn't seem like an unreasonble change, and I'd rather be consistent with conventions. I'll make the change and see if anything breaks.